projects/lights_control/inc/lights_config.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include "can.h"
+
+#include "lights_events.h"
+
+const GPIOAddress LIGHTS_CAN_RX_ADDR = { GPIO_PORT_B,
```

Comment:
Looks like you forgot some semicolons? Also I don't think this will work since everywhere the header is included these will be redefined. You should either make them macros or move them to an internal `*.c` file and get them via accessors. Alternatively you can use `extern` but this is highly discouraged.

If this were C++ this would be okay in a `namespace` with `constexpr` but this is C.

Reply:
Which one is recommended?
1. Macros
2. Having a `lights_config.c` file which will initialize these config variables with an exposed `lights_initialize_modules()` function that initializes all the lights modules, and gets called from main.
I believe at this moment this file is only going to be included in `main.c` because it only contains initialization variables, so I might not have the redefinition problem, but that seems both bad practice and limiting since in the future I wouldn't be able to include this header file in any module in case I need some of its variables.